Rakefile

```diff
@@ -60,7 +60,7 @@ namespace :generate do
 
       # Rails 4 cannot use a `rails` binstub generated by Bundler
       sh "rm -f #{bindir}/rails"
-      sh "bundle exec rails new ./tmp/example_app --no-rc --skip-javascript --skip-sprockets --skip-git --skip-test-unit --skip-listen --skip-bundle --template=example_app_generator/generate_app.rb"
+      sh "bundle exec rails new ./tmp/example_app --no-rc --skip-javascript --skip-git --skip-test-unit --skip-listen --skip-bundle --template=example_app_generator/generate_app.rb"
```

Comment:
Could you explain why you changed this?

This has a cost. It seems that sprockets adds sass gem. On Ruby 1.9.3 it adds deprecation warning and breaks the CI

```
     Failure/Error:
       expect(capture_exec(custom_env, exec_script)).to eq(
         "#{::Rails.root}/spec/mailers/previews"
       )
     
       expected: "/home/travis/build/rspec/rspec-rails/tmp/example_app/spec/mailers/previews"
            got: #<struct CaptureExec io="DEPRECATION WARNING:\nSass 3.5 will no longer support Ruby 1.9.3.\nPlease up...ible.\n\n/home/travis/build/rspec/rspec-rails/tmp/example_app/spec/mailers/previews", exit_status=0>
```

I revert this line and test on Ruby 1.9.3 and 2.5.3 without issue. I didn't test on all rails version. But maybe it doesn't need to be committed?

Reply:
Yep I'd like this reverted too